aspnetcore/signalr/groups.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Manage users and groups in SignalR
+author: 
+description: 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: 
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/groups
+---
+
+# Manage users and groups in SignalR
+
+## Users?
+
+A user is one or more connections associated with an identifier. By default SignalR will use the `ClaimTypes.NameIdentifier` from the `ClaimsPrincipal` associated with the connection. There can be multiple connections for a single user. For example you can be connected on your desktop, your phone, and your laptop. If you send to the user all three connections will receive the message.
+
+You can send a message to a specific user by passing the user identifier to the `User(...)` function in your hub method as shown in the following example. One important aspect to note is that the user identifier is case-sensitive.
+
+```csharp
+public Task SendPrivateMessage(string user, string message)
+{
+    return Clients.User(user).SendAsync("ReceiveMessage", message);
+}
+```
+
+The user identifier can be customized by creating your own `IUserIdProvider`.
```

Comment:
The user identifier can be customized by creating your own `IUserIdProvider`, and registering it in `ConfigureServices`